mee/clock.h

```diff
@@ -14,10 +14,17 @@ struct __mee_clock_vtable {
     long (*set_rate_hz)(struct mee_clock *clk, long hz);
 };
 
+typedef void (*mee_clock_pre_rate_change_callback)(void *priv);
 typedef void (*mee_clock_rate_change_callback)(void *priv);
```

Comment:
Can you rename these "post_rate_change", to keep it uniform with the "pre_rate_change"?